neural_speed/models/grok/grok.h

```diff
@@ -0,0 +1,53 @@
+//  Copyright (c) 2024 Intel Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#ifndef GROK_H
+#define GROK_H
+
+#include "models/model_utils/model_files.h"
+#include "models/model_utils/model_types.h"
+
+enum grok_model {
+  GROK_314B,
+};
+
+static const model_scratch grok_mem_req(int n_layers, float enlarge_scale = 1.0f) {
+  switch (n_layers) {
+    case 64:
+      return {
+          static_cast<unsigned long long>(enlarge_scale * 3072) * MB,
```

Comment:
static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
          static_cast<unsigned long long>(scratch_size_ratio * 2048) * MB,
          static_cast<unsigned long long>(scratch_size_ratio * 4096 * 10) * MB,

can we use this policy to keep the same as other models?

Reply:
What's mean?

Reply:
The second scratch memory is less than the first and third temporary memories.

Just make sure the 2nd one uses the least and change the 1st and 3rd from 3072 to 4096, keeping the same as other models.